doc/nrf/ug_thread_supported_features.rst

```diff
@@ -72,3 +72,15 @@ Developers can make use of the OpenThread API to obtain information about the li
 Enhanced-ACK probing is the most power efficient method to retrieve link metrics since very little message overhead is required.
 |NCS| provides full Link Metrics support even for the RCP architecture, which is the most technically challenging one since the radio driver must handle the injection of Information Elements on time to match the acknowledgment timing requirements.
 The decision on how to interpret the link metrics information to adjust the transmission power is left to the application itself.
+
+Thread Domain Unicast Addressing
+================================
+
+Thread 1.2 Specification introduces concept of Thread Domains.
+
+A Thread Domain is a set of Thread Devices that receive and apply common Thread Domain operational configuration. The Thread Domain operational configuration enables
+Thread Devices to join and participate in larger interconnected systems extending beyond the limits of a single Thread Network. A user or network administrator may use functions
+of Thread Commissioning or of Thread Border Routers to set up a common Thread Domain operational configuration for Thread Devices belonging to different Thread
+Networks or Partitions having potentially different per-network credentials.
```

Comment:
```suggestion
A Thread Domain is a set of Thread Devices that receive and apply a common Thread Domain operational configuration.
The Thread Domain operational configuration enables Thread Devices to join and participate in larger interconnected systems extending beyond the limits of a single Thread Network. 
A user or network administrator may use functions of either Thread Commissioning or Thread Border Routers to set up a common Thread Domain operational configuration for Thread Devices.
The Thread Devices can belong to different Thread Networks or Partitions that have potentially different per-network credentials.
```
I am not sure, but I feel the word "common" could be replaced here with "shared" in both instances.

Also, the text isn't clear, but does a Thread Domain have to extend beyond a single Thread Network?